lib/fx/configuration.rb

```diff
@@ -40,9 +40,25 @@ class Configuration
     # @return Boolean
     attr_accessor :dump_functions_at_beginning_of_schema
 
+    # Check function bodies during creation by issuing
+    # `SET LOCAL check_function_bodies TO <<value>>;` before creating functions.
+    #
+    # This is only useful with the Postgresql adapter.
+    #
+    # See https://stackoverflow.com/a/36983831 and
+    # https://www.postgresql.org/docs/9.5/runtime-config-client.html#GUC-CHECK-FUNCTION-BODIES
+    #
+    # Set to `nil` to use the database default configuration.
+    # The default and recommended value is `false` to mimic pg_dump's behavior.
```

Comment:
I am not sure about this. The [docs](https://www.postgresql.org/docs/9.5/runtime-config-client.html#GUC-CHECK-FUNCTION-BODIES) say:
> This parameter is normally on.

So maybe the default should be `true` here, with the possibility to set it to `false` for specific use cases?

Reply:
You could do that. It's off by default for `pg_dump` from what I remember, and this gem is the closest thing to `pg_dump` so it seemed reasonable to follow the `pg_dump` conventions. Either way works. Just having the option would be a huge win!

Reply:
See: https://dba.stackexchange.com/questions/284260/how-to-prevent-pg-dump-from-generating-set-default-table-access-method-heap -- it shows that `false` the default value for `pg_dump`.